src/components/gds-logo-grid/gds-logo-grid.scss

```diff
@@ -0,0 +1,57 @@
+// TODO: Move styles to _index.css if needed for WP?
+@use "index" as *;
+
+:host {
+  @include base;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 263px);
+  grid-template-rows: repeat(auto-fill, 263px);
+  grid-gap: 0;
+  justify-content: center;
+  max-width: 1060px;
+}
+
+::slotted(gds-logo-grid-item) {
+  border-top: 1px solid #eeeeee;
+  border-left: 1px solid #eeeeee;
+}
```

Comment:
So we add a slight border in between each element in the grid?

Reply:
Thanks! :) 

Yes, it's the logo grid has borders in between.
https://xd.adobe.com/view/19d171d1-38f9-43e7-6556-de035c26e07a-6fe0/screen/b6f5e00f-812e-49e4-8ecf-d35463aaf9b3/
<img width="1338" alt="image" src="https://user-images.githubusercontent.com/1070946/85266075-fe617b80-b47b-11ea-9834-dd4368eea766.png">

Reply:
suppose eventually it will be
```css
border-top-width: var(--grid-border-width-or-some-other-cool-variable-name);
```

but this looks exactly like design so it's perfect